pkg/server/syncHandler.go

```diff
@@ -27,6 +29,7 @@ func (s *ServerConfig) SyncResources(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusBadRequest)
 		return
 	}
+	metrics.RequestSize.Observe(float64(len(syncEvent.AddResources) + len(syncEvent.UpdateResources) + len(syncEvent.DeleteResources)))
```

Comment:
NOTE: this metric isn't collected from the middleware because we need to decode the request body (SyncEvent)